obal/data/roles/setup_sources/defaults/main.yaml

```diff
@@ -1,2 +1,3 @@
 ---
 setup_sources_annex_options: ""
+setup_sources_annex_workaround: false
```

Comment:
Why would we want this work around to be configurable?

Reply:
I initially thought to make this configurable because it does not happen on the epel version of git-annex that we use on the foreman infra.

But then every user would have to enable that themselves, so you're right and I should drop the flag again.

Reply:
dropped